rosalind_ini4.py

```diff
@@ -1,17 +1,4 @@
 def main(input):
-    a, b = input.split()
-    a = int(a)
-    b = int(b)
-    
-    sum = 0
-	
-    if a % 2 == 0:
-        a += 1
-    else:
-        pass
-	
-    while a <= b:
-        sum += a
-        a += 2
-		
-    return sum
\ No newline at end of file
+    a, b = map(int, input.split())
+
+    return sum([i for i in range(a, b + 1) if i % 2])
```

Comment:
In other words, return the sum of this list of integers, which is made of the numbers between `a` and `b + 1` that when divided by two, return a value of `1`.